docs/ext/commands/api.rst

```diff
@@ -1095,12 +1095,12 @@ Exception Hierarchy
 Warnings
 ----------
 
-.. autoclass:: disnake.ext.commands.MessageContentPrefixWarning
+.. autoclass:: MessageContentPrefixWarning
 
 Warning Hierarchy
 ~~~~~~~~~~~~~~~~~~~
 
 .. exception_hierarchy::
 
-    - :class:`DiscordWarning`
+    - :class:`~.DiscordWarning`
         - :class:`~.commands.MessageContentPrefixWarning`
```

Comment:
I believe this should be 
```suggestion
        - :class:`~.MessageContentPrefixWarning`
```

to fit in with the rest of the changes

Reply:
Didn't have any impact except for the aesthetics, perhaps. Thanks!